prdoc/pr_5327.prdoc

```diff
@@ -0,0 +1,39 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "Add `get_preset` to the testnet runtimes"
```

Comment:
nit: maybe sth like: "Moved presets to the testnet runtimes" ?
Would be more informative?

Reply:
@michalkucharczyk fixed: https://github.com/paritytech/polkadot-sdk/pull/5327/commits/e8f3bbbb51f17c598ffc3718cdf8e52a638047c3